ssh/tunnel.py

```diff
@@ -81,6 +82,9 @@ def tunnel(user: str, key_path: str, host: str, port: int=22):
     logger.debug('Closing SSH Tunnel: ' + ' '.join(close_tunnel))
     check_call(close_tunnel)
 
+    if os.path.exists(tunnel_socket.name):
+        os.remove(tunnel_socket.name)
```

Comment:
A comment explaining why we're not cleaning up the file, and getting rid of this `os.remove()`, works for me. 

Ideally the code that's responsible for creating the file ought to be deleting it. If we could create a tmp filename and pass that to SSH and let it own the lifecycle of the file, that'd be ideal. We could append a random string to `tempfile.gettempdir()` and let SSH own the file entirely.